src/app/components/feedback/feedback.component.ts

```diff
@@ -19,12 +21,15 @@ export class FeedbackComponent implements OnInit {
     ])
   })
 
-  constructor(private feedbackService: FeedbackService, private matSnackBar: MatSnackBar) {
+  constructor(private feedbackService: FeedbackService, private matSnackBar: MatSnackBar, private router: Router) {
   }
 
   ngOnInit(): void {
     this.loadFeedback();
     this.userProfile = JSON.parse(sessionStorage.getItem('loggedInUser') || '{}');
+    if (this.userProfile === null || this.userProfile === undefined){
```

Comment:
create a condition to open google sign in dialog when google sign in data doesnt exsists in the user browser window